src/webgl/light.js

```diff
@@ -409,8 +409,20 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
  */
 p5.prototype.lights = function() {
   this._assert3d('lights');
-  this.ambientLight(128, 128, 128);
-  this.directionalLight(128, 128, 128, 0, 0, -1);
+  if (this._colorMode !== 'rgb') {
```

Comment:
could you update this to `constants.RGB`? here's an example of how it's done by importing constants and then calling it in [this file](https://github.com/processing/p5.js/blob/bb381febc246c7599381801ae314b3d40330d9d8/src/webgl/3d_primitives.js#L1078)

Reply:
same thing on line 416